docs/Learning-Environment-Design-Agents.md

```diff
@@ -14,57 +15,53 @@ action, for example, move the agent in one direction or another. In order to
 your agent must calculate a reward value at each action. The reward is used to
 discover the optimal decision-making policy.
 
-The Policy class abstracts out the decision making logic from the Agent itself so
+The `Policy` class abstracts out the decision making logic from the Agent itself so
 that you can use the same Policy in multiple Agents. How a Policy makes its
-decisions depends on the kind of Policy it is. You can change the Policy of an
-Agent by changing its `Behavior Parameters`. If you set `Behavior Type` to
-`Heuristic Only`, the Agent will use its `Heuristic()` method to make decisions
-which can allow you to control the Agent manually or write your own Policy. If
-the Agent has a `Model` file, it Policy will use the neural network `Model` to
-take decisions.
+decisions depends on the `Behavior Parameters` associated with the agent. If you
+set `Behavior Type` to `Heuristic Only`, the Agent will use its `Heuristic()`
+method to make decisions which can allow you to control the Agent manually or
+write your own Policy. If the Agent has a `Model` file, it Policy will use
+the neural network `Model` to take decisions.
 
 ## Decisions
 
 The observation-decision-action-reward cycle repeats each time the Agent request
 a decision.
 Agents will request a decision when `Agent.RequestDecision()` is called. If you need
 the Agent to request decisions on its own at regular intervals, add a
-`Decision Requester` component to the Agent's Game Object. Making decisions at regular step
+`Decision Requester` component to the Agent's GameObject. Making decisions at regular step
 intervals is generally most appropriate for physics-based simulations. For example, an
 agent in a robotic simulator that must provide fine-control of joint torques
 should make its decisions every step of the simulation. On the other hand, an
 agent that only needs to make decisions when certain game or simulation events
-occur, should call `Agent.RequestDecision()` manually.
+occur, such as in a turn-based game, should call `Agent.RequestDecision()` manually.
 
-## Observations
+## Observations and Sensors
 
-To make decisions, an agent must observe its environment in order to infer the
-state of the world. A state observation can take the following forms:
+To make informed decisions, an agent must first make observations of the stete of
```

Comment:
```suggestion
To make informed decisions, an agent must first make observations of the state of
```
Can you run a spell checker on the documentation to make sure there are no other typos?

Reply:
Good catch. Just fixed this and a few other spelling/grammatical errors.